sdks/java/core/src/test/java/org/apache/beam/sdk/io/AvroIOTest.java

```diff
@@ -261,7 +262,9 @@ public void testWriteDisplayData() {
 
     private enum WriteMethod {
       AVROIO_WRITE,
-      AVROIO_SINK
+      AVROIO_SINK,
+      @Deprecated
+      AVROIO_SINK_WITH_FORMATTER
```

Comment:
Thanks for taking care of this deprecation, you have to add a comment `/** @deprecated ....*/` with the reason to make checkstyle happy